Sources/NIO/System.swift

```diff
@@ -112,6 +112,11 @@ private func assertIsNotBlacklistedErrno(err: CInt, where function: StaticString
     assert(!isBlacklistedErrno(err), "blacklisted errno \(err) \(String(cString: strerror(err)!)) in \(function))")
```

Comment:
I think we can now delete the `assertIsNotBlacklistedErrno` function, right?

Reply:
Since it is no longer used, I guess yes it could "just" be deleted...